docs/RH-SSO/ServiceDefinition.md

```diff
@@ -0,0 +1,123 @@
+# BC Government Single Sign-On Service Definition
+
+## Service Description
+
+### Summary
+
+The BC Government **Single Sign-On (SSO)** simplifies authentication and authorization by enabling development teams to secure applications.
+
+Development teams will be provisioned a private `realm` hosted on the [OpenShift Container Platform (OCP)](../OCP/ServiceDefinition.md) that can be self-administered and customized to the needs of one or more projects.
+
+Each realm is configured out-of-the-box to support BCeID and IDIR as well as other 3rd-party oAuth 2.0 providers.
+
+### Features & Functions
+
+Users of this service gain access to the following:
+
+#### Private Realm(s)
+
+Each development team is given an SSO instances correlating to each of their environments (dev, test, and prod). These instances, commonly known as a `realm`, come configured out-of-the-box with following authentication providers:
+
+* BC Government **IDIR**
+* [BCeID](https://www.bceid.ca)
+* [GitHub](https://developer.github.com/apps/building-oauth-apps/authorizing-oauth-apps/)
+
+#### Access
+
+Each realm has an its own web interface that development teams can use for self-serve administration. To integrate with your Web application the following industry standard protocols are available:
+
+ * OpenID Connect
+ * OAuth 2.0
+ * SAML 2.0
+
+For native mobile apps, the BC Government DevHub produced the following SDKs:
+
+ * [iOS SDK](https://github.com/bcgov/mobile-authentication-ios)
+ * [Android SDK](https://github.com/bcgov/mobile-authentication-android)
+
+
+#### Community
+
+We encourage a vibrant community around SSO that can be found on the `#sso` channel in [RocketChat](https://reggie.pathfinder.gov.bc.ca/?intention=LOGIN#error=login_required). Use your GitHub ID or IDIR to sign-up and join.
+
+
+### Eligibility & Prerequisites
+
+This service collection is offered to BC Government development teams building cloud native web or mobile application on the [OpenShift Container Platform](../OCP/ServiceDefinition.md).
+
+
+### How to Request
+
+Once a development team has their dev, test and prod environments setup on the OCP they can request their realm. Send the information noted in the [SSO Realm Request](https://pathfinder-faq-ocio-pathfinder-prod.pathfinder.gov.bc.ca/OCP/RequestSSORealm.html) to the `#devops-requests` channel of [RocketChat](https://reggie.pathfinder.gov.bc.ca/?intention=LOGIN#error=login_required).
```

Comment:
might stop using the wiki page soon, could use the devhub link instead" https://developer.gov.bc.ca/Authentication-and-Authorization/SSO-Realm-Creation